sdk/appcenter/src/test/java/com/microsoft/appcenter/AppCenterTest.java

```diff
@@ -200,6 +198,27 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
 
         /* Mock network state helper. */
         when(NetworkStateHelper.getSharedInstance(any(Context.class))).thenReturn(mNetworkStateHelper);
+
+        /* Mock string splitter. */
+        whenNew(TextUtils.SimpleStringSplitter.class).withAnyArguments().then(new Answer<TextUtils.SimpleStringSplitter>() {
```

Comment:
I pushed a new implementation using pure java instead of Android string splitter here: feature/transmission-token-parsing. That way there is no need for mocking. Can you please take a look and merge it to this branch ?

Reply:
I should have told you about this implementation before, it was lying down in a local branch for more than a week :sweat_smile:.

Reply:
These PRs are targeted to different branches. I think we need to merge some of them before merging the changes between them.

Reply:
True :/.